.gitignore

```diff
@@ -0,0 +1 @@
+app
```

Comment:
Why add this?

Reply:
For testing, I use a mount : 
```
docker run -i --entrypoint="/bin/bash" -v ${PWD}/app:/tmp/build/put/app opendoor/concourse-helm-resource:dev -c '/opt/resource/out /tmp/build/put' < ~/Downloads/serviceregistry-test-helm-failure-payload.json
```
No biggie for me to remove it ... its only for testing and should be OK with our without it